src/shared-components/dialogModal/index.tsx

```diff
@@ -11,13 +12,18 @@ import {
   ModalTitle,
   CrossIconContainer,
 } from './style';
+import { Colors, primaryTheme, ThemeType } from '../../constants';
 
 export interface DialogModalProps {
   /**
    * Dialog Modal content.
    * Must contain at least 1 button and is responsible for closing the modal.
    */
   children: React.ReactNode;
+  /**
+   * DialogModal background color. Defaults to the current theme's `white` if not specified.
+   */
+  modalColor?: Colors['background'];
```

Comment:
Since we're passing the prop to a modal component--and eventually pass it as a prop named `backgroundColor `--should we call this `backgroundColor`? I think that is our convention in most other components, too.